benchmarks/benchmark/tools/profile-generator/container/benchmark_serving.py

```diff
@@ -572,6 +676,17 @@ def print_and_save_result(args: argparse.Namespace, benchmark_duration, total_re
   benchmark_result['total_tokens'] = int(total_tokens)
   benchmark_result['tokens_per_min'] = tokens_per_min
 
+  if args.stream_request:
+    mean_ttft=np.mean(ttfts)
+    median_ttft=np.median(ttfts)
+    p99_ttft=np.percentile(ttfts, 99)
+    print(f"Mean TTFT (s): {mean_ttft:.2f}")
+    print(f"Median TTFT (s): {median_ttft:.2f}")
+    print(f"P99 TTFT (s): {p99_ttft:.2f}")
+    benchmark_result['mean_ttft'] = mean_ttft
+    benchmark_result['median_ttft'] = median_ttft
+    benchmark_result['p99_ttft'] = p99_ttft
```

Comment:
nit, why not use `get_stats_for_set()`

Reply:
No particular reason. Changed it to use `get_stats_for_set()` instead.